cmd/machine-config-daemon/pivot.go

```diff
@@ -366,10 +364,7 @@ func run(_ *cobra.Command, args []string) error {
 	}
 
 	if !changed {
-		glog.Info("Already at target pivot; exiting...")
-		if exit77 {
-			os.Exit(77)
-		}
+		glog.Info("Already at target oscontainer")
 	} else if reboot || utils.FileExists(runPivotRebootFile) {
```

Comment:
debugging the very latest weirdness I've witnessed with pivot, it would be nice to log if we rebooted cause of the flag or the file